packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -41,7 +41,11 @@ class PopupMenuDivider extends PopupMenuEntry<dynamic> {
 
   final double height;
 
-  Widget build(BuildContext context) => new Divider(height: height);
+  _PopupMenuDividerState createState() => new _PopupMenuDividerState();
+}
+
+class _PopupMenuDividerState extends State<PopupMenuDivider> {
+  Widget build(BuildContext context) => new Divider(height: config.height);
```

Comment:
Well I guess the PopupMenuEntryState would be a bit more complex. It has to introduce the InkWell. The onTap handler would be in there too.